Resources/translations/messages.fr.xliff

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
```

Comment:
then just this line in the header of your templates : 

    {% trans_default_domain 'PrestaImageBundle' %}

ex : https://github.com/sonata-project/SonataAdminBundle/tree/3.x/Resources/translations